src/systemd_ctypes/bus.py

```diff
@@ -31,6 +32,13 @@
 logger = logging.getLogger(__name__)
 
 
+class Name(enum.IntFlag):
```

Comment:
I think the namespacing is incorrect here.  It exists at the same level as classes with the names `Bus` and `BusMessage`.  This either needs to:

 - be called `BusName` (actually, in retrospect, I like `BusNameFlags`)
 - be moved under `class Bus:` and then it can be called `Name` (but please then rename to `NameFlags`).

Reply:
Right, this makes  a lot more sense. I like the second one more, that's what I implemented.